src/lsm/binary_search.zig

```diff
@@ -164,6 +198,40 @@ const test_binary_search = struct {
             try std.testing.expectEqual(expect.exact, actual.exact);
         }
     }
+
+    fn random_search(random: std.rand.Random, iter: usize) !void {
+        const keys_count = @floatToInt(usize, @trunc(random.floatExp(f64) * @intToFloat(f64, iter)));
```

Comment:
Could we extract this to a function (from here and the other fuzzers that use it) into a `test/fuzz.zig`? And include a `max` parameter to keep a sane limit for the unit tests' memory requirements. (It is fine if that is the only function there for now; we can fill it out over time).

If a fuzzer OOMs I want it to be because of a bug, not a bad roll of the RNG. :sweat_smile: